app/routes/events/components/EventAdministrate/Allergies.tsx

```diff
@@ -81,27 +102,58 @@ const Allergies = ({ eventId, event, error, loading, registered }: Props) => {
       sorter: (a, b) => a.user.allergies.localeCompare(b.user.allergies),
     },
   ];
+
+  const columns = event.feedbackRequired
+    ? initialColumns.concat({
+        title: 'Tilbakemelding',
+        dataIndex: 'feedback',
+        centered: false,
+        render: (feedback) => <span>{feedback || '-'}</span>,
+        sorter: (a, b) => a.feedback.localeCompare(b.feedback),
+      })
+    : initialColumns;
+
   const numOfAllergies = () => {
     return registered.filter(
-      (registration) => registration.user.allergies.length != 0
+      (registration) => registration.user.allergies?.length != 0
     ).length;
   };
   return (
     <div>
-      <Flex column>
-        {numOfAllergies() == 0 ? (
-          <li>Ingen påmeldte med allergier</li>
-        ) : (
-          <Table
-            hasMore={false}
-            columns={columns}
-            loading={loading}
-            data={registered.filter((registration) => {
-              return registration.user.allergies;
-            })}
-          />
-        )}
-      </Flex>
+      {currentUser.id &&
+      (event.createdBy || event.createdBy.id) &&
+      (currentUser.id === event.createdBy || // this is needed for testing
```

Comment:
👀

Reply:
I have now fixed up in the types and it looks a lot cleaner